src/richie/apps/search/indexers/categories.py

```diff
@@ -101,6 +106,12 @@ def get_es_document_for_category(cls, category, index=None, action="index"):
         # Shorcut to the category's page node
         node = category.extended_object.node
 
+        # Find the meta category the current category falls under
+        try:
+            kind = category.get_meta_category().extended_object.reverse_id
+        except IndexError:
```

Comment:
```suggestion
        except Category.DoesNotExit:
```

+ add comment
`
The category as no meta which means it is a meta itself. Let's give it a None kind.
`